### PR TITLE
Flight Plan Parcel Linked Table

### PR DESCRIPTION
**Pull request by ServiceDog**
Adds flight_plan_parcel linked table

Updates parcel record:
- Removes itinerary_id
- Adds user_id (needed now that itinerary_id is gone, no more link to user)
- Adds weight_g

### CODE REVIEW

File: proto/svc-storage-grpc-parcel.proto [hunk 1]
```diff
@@ -56,8 +56,11 @@ enum ParcelStatus {
 
 // Data struct with parcel data
 message Data {
-    // the itinerary UUID associated with the parcel
-    string itinerary_id = 1;
+    // the user UUID associated with this parcel
+    string user_id = 1;
+
+    // the weight (in grams)
+    uint32 weight_g = 2;
```
Comment (owlot) [inline proto/svc-storage-grpc-parcel.proto:63]: We decided to go with the full unit name so there can be no confusion as to what the abbreviation means. So should be named `weight_grams`

Comment (ServiceDog) [reply proto/svc-storage-grpc-parcel.proto]: https://github.com/Arrow-air/svc-storage/pull/76/commits/73d3136188511e164c0e28f52e3e373b1e657558

File: proto/svc-storage-grpc-parcel.proto
Comment (owlot) [inline proto/svc-storage-grpc-parcel.proto:1]: I think your vscode is not reading the .editorconfig file.
Do you have the extension https://open-vsx.org/extension/EditorConfig/EditorConfig installed?

**Comment by ServiceDog** [comment]
Implemented by https://github.com/Arrow-air/svc-storage/pull/80#event-9804426391

**Pull request closed without merging**